projects/storefrontlib/shared/components/split-view/view/view.component.ts

```diff
@@ -73,6 +83,15 @@ export class ViewComponent implements OnInit, OnDestroy {
     protected cd: ChangeDetectorRef
   ) {}
 
+  @Optional()
```

Comment:
Lets keep our injectors at the top of the class to be consistent.

Reply:
I'll keep this in mind ;)